ci/doc.yml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   - pip
   - python>=3.8
-  - xarray>=0.20.2
+  - xarray>=2022.05.0.dev0
```

Comment:
it''s my understanding that the latest pre-release isn't published on conda-forge but on PyPI. Should it be put under the `pip` section 

```yaml
...
- pip:
  - xarray>=2022.6.0rc0
```